Source/ACE.Server/WorldObjects/PressurePlate.cs

```diff
@@ -72,5 +77,17 @@ public override void OnActivate(WorldObject activator)
 
             base.OnActivate(activator);
         }
+
+        public override void ActOnUse(WorldObject activator)
+        {
+            // Pressure Plate should be activating a linked object configured via landblock_link
```

Comment:
if the data is being fixed for this, then PressurePlate.ActOnUse() should just be removed entirely, and just base.ActOnUse() for the error message